shared/js/smart-screen/card_manager.js

```diff
@@ -711,6 +711,18 @@
       return found;
     },
 
+    findContainingFolder: function cm_findContainingFolder(query) {
+      var folder;
+      this._cardList.some(function(card) {
+        if (card instanceof Folder) {
+          if (card.findCard(query)) {
+            folder = card;
+          }
+        }
+      });
+      return folder;
+    },
+
```

Comment:
This function is duplicated with part of the latest `card_manager::removeCard`
We may need to merge them as one.